### PR TITLE
Fix 400 - Bad Request when updating user attributes in non OAuth applications

### PR DESCRIPTION
**Pull request by nipunsampath**
### Purpose
The issues https://github.com/wso2/product-is/issues/17746 and https://github.com/wso2/product-is/issues/17748 occurs due to an OIDC protocol update request that is being sent when updating the user attributes.  For non OAuth applications, this request returns `400 - Bad Request`.

This PR modified the logic so that the OIDC protocol update request is only sent for OAuth applications.

### Related Issues
- https://github.com/wso2/product-is/issues/17748
- https://github.com/wso2/product-is/issues/17746

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx [hunk 1]
```diff
@@ -1079,10 +1079,14 @@ export const AttributeSettings: FunctionComponent<AttributeSettingsPropsInterfac
...
-
+        
+        const isProtocolOAuth: boolean = technology?.find((protocol: InboundProtocolListItemInterface) => 
+            protocol.type === SupportedAuthProtocolTypes.OAUTH2) !== undefined;
+        
         Promise.all([
             updateClaimConfiguration(appId, submitValue),
-            updateAuthProtocolConfig<OIDCDataInterface>(appId, oidcSubmitValue, SupportedAuthProtocolTypes.OIDC)
+            isProtocolOAuth ? updateAuthProtocolConfig<OIDCDataInterface>(appId, oidcSubmitValue, 
+                SupportedAuthProtocolTypes.OIDC) : Promise.resolve()
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx:1089]:
```suggestion
            isProtocolOAuth 
              ? updateAuthProtocolConfig<OIDCDataInterface>(appId, oidcSubmitValue, 
                SupportedAuthProtocolTypes.OIDC) 
              : Promise.resolve()
```

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx [hunk 2]
```diff
@@ -1079,10 +1079,14 @@ export const AttributeSettings: FunctionComponent<AttributeSettingsPropsInterfac
         if (applicationConfig.excludeSubjectClaim && onlyOIDCConfigured) {
             delete submitValue.claimConfiguration.subject;
         }
-
+        
+        const isProtocolOAuth: boolean = technology?.find((protocol: InboundProtocolListItemInterface) => 
+            protocol.type === SupportedAuthProtocolTypes.OAUTH2) !== undefined;
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx:1084]:
```suggestion
        const isProtocolOAuth: boolean = !!technology?.find((protocol: InboundProtocolListItemInterface) => 
            protocol.type === SupportedAuthProtocolTypes.OAUTH2);
```

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx (hunk 1 above)
Comment (nipunsampath) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx]: Fixed with 6170570b3e4ac92ee2e3ec8d8fd8f8dbfa0acebc

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx (hunk 2 above)
Comment (nipunsampath) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx]: Fixed with 6170570b3e4ac92ee2e3ec8d8fd8f8dbfa0acebc

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**